codecov.yml

```diff
@@ -0,0 +1,32 @@
+codecov:
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
```

Comment:
We can easily turn off GitHub comments. What's everyone's thought on keeping or removing them from PRs? There should be a way to easily open the full report in Codecov from the PR checks section. They may not show up for this PR since we don't have any coverage reporting in `main` yet.